app/src/main/res/xml/root_preferences.xml

```diff
@@ -8,4 +9,9 @@
             app:title="@string/theme"
             app:useSimpleSummaryProvider="true" />
     </PreferenceCategory>
+    <CheckBoxPreference
+        android:defaultValue="false"
+        android:key="show_notifications"
+        android:summary="for debugging, independent of TRIfA"
+        android:title="Show notifications" />
```

Comment:
can you make that translate able?

Reply:
okey. moved those two strings to strings.xml